static/docs/command-reference/pipeline/show.md

```diff
@@ -52,6 +51,29 @@ instead of stages.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+## Paging the output
+
+This command's output is automatically pushed to `less` command line tool if
+`less` is runnable (installed and can be executed in your shell). To be exact,
+the actual command is `less --chop-long-lines --clear-screen`.
+
+If `less` is not runnable (usually Microsoft Windows), the ouput is simply
+printed out.
+
+> Note Microsoft Windows users may also want to read
+> [this how-to](/doc/user-guide/running-dvc-on-windows#enabling-paging-with-less).
+
+### Providing a custom pager
+
+You can also override the default pager via `DVC_PAGER` environemnt variable.
+
+For example
+
+```
+# one-time change in Bash
```

Comment:
don't put it as a comment, expand and move it to the human-readable section above. This is comment is not supposed to be carried over to the .bash_profile, right?

Reply:
promoted the comment to a paragraph, thanks!



> This is comment is not supposed to be carried over to the .bash_profile, right?

.bash_profile requires `export`